content/07.pathogenesis.md

```diff
@@ -46,8 +46,12 @@ One severe example includes reports of cytokine storm-like responses in patients
 #### Reproduction Number and Dynamics of Transmission
 
 Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
-The basic reproduction number, R~0~, is the expected number of new infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
+In models the quantities of basic reproduction (or reproductive) number and the timescale of infection have a large influence on the population level dynamics of epidemics.
+The basic reproduction number, R~0~, is the expected number of new (secondary) infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
+R~0~ is a unitless number which in models is a combination of parameters principally related to the transmission rate (rate of infection transmitting interactions for an infectious individual) and the infectious period.
+When R~0~ is greater than 1, there is potential for an epidemic to grow.
```

Comment:
```suggestion
When R~0~ is greater than 1, the number of infected individuals grows exponentially.
```

Reply:
I put "potential" because I meant that some areas may not have any infectious individual (or object) introduced yet but their qualities may allow for an epidemic to grow if an infectious individual(s) is introduced (may require multiple times due to stochasticity). Also I think R0 is a characteristic of the population regardless, even if R changes; meaning sometimes R0 can be greater than 1 but R isn't?  I think this is why when people talk about R0 & R it can get confusing to people outside the field -- using formulas maybe makes it easier to follow. But what you corrected I think may be the more common statement & maybe "exponential" does account for the case of the unstable zero equilibrium (since 0 growing exponentially doesn't make sense -- I'd have to look up the theoretical math definitions again!). I'm writing this sort of quickly but just wanted to include a bit of a response. This will take some time to get to clearer additions & if I add more then I will try better to stay within a clear structure.

Reply:
Thanks @yemarshall I get your point, i.e. "potential" to grow to account for noisy introduction of virus & small pop sizes. however in the specific case of this covid pandemic, with many large infected populations, R>1 means almost certainly that # infected will grow, and my fear including "potential" here is that lay reader will put it in the place of "almost certainly", which is not the case.

Reply:
good point, I'm going to think about this a bit more too.

Reply:
Non-expert here -- would something like "an epidemic will grow, with the potential for exponential growth" cover both concerns?

Reply:
@alavendelm How about: "If R~0~ > 1, a pathogen can invade." OR "If R~0~ > 1, a pathogen can invade and start an epidemic with exponential growth of the number of infected individuals." OR I even also considered removing this sentence altogether and just keeping the Rt > 1 or Rt < 1 sentence. These are closer to the definition, & they account for the case where R0>1 but Rt<1 (e.g. with interventions) (?? note that I've worked on mechanistic models & largely avoided this terminology). I understand trying to avoid misinterpretations & to make things clear, but we also need to make sure that we don't simplify so much that it can cause other misunderstanding? In the end, from what I've read for other diseases, sometimes in obvious cases people see when the science works & when it doesn't though (1) they may not consider the uncertainty & variation and (2) they sometimes forget or don't pay attention to history (unless reminded? or unless it's essential/current?) -- this is way more complicated a topic than what I know well but I do think that people like it when science/medicine/whatever works to help them in obvious way.

Reply:
One other thing to consider from a non-expert perspective: The way R0 and Rt are currently described it sort of sounds like they are a property of the disease separate from the population/epidemic and that depending on "the reproduction number" the epidemic will grow or shrink or change in some way... the reproduction number could also be thought of as simply a "description" of how the epidemic is growing or shrinking or changing in some way. 

Especially when Rt is calculated from population level statistics, it's not that "when Rt is greater than 1 an epidemic has the potential to grow" it's that "Rt greater than 1, by definition, means an epidemic is growing".

Reply:
Agree with comment from @rdvelazquez this is an important distinction and the latter phrasing captures it well

> Rt greater than 1, by definition, means an epidemic is growing

Reply:
```suggestion
```
What if we just delete this sentence? The sentence in line 54 seems to cover it well enough.

Reply:
from @yemarshall:

> On thinking about it again, it seems my suggestion may be lax to intentionally skip on explaining what R0<1 or R0>1 b/c people see it everywhere & so it's useful to provide an explanation that's both correct and clear. Since we also include what R>1 and R<1 means then we don't have to worry as much about people misinterpreting R0<1 or R0>1. Either we put what we have, or I can put another suggestion which makes it even more clear while also being correct and including making difference between R0 & Rt even clearer if needed -- among all the discussion there are at least a couple of suggestions which cover everything well enough (& this along with other stuff i wrote address Question4 in #313 ). If needed I can also / will just settle on the final strategy I found for doing these things to reduce so much discussions -- I'll put as close as possible to textbook definitions or mathematically exact descriptions with references & which are as clear as possible.

Your reasoning for not wanting to skip explaining what R0 > 1 means makes a lot of sense. I think the explanation will be closely related to how we explain the differences between R0 and Rt... @alavendelm are you planning to address the differences between R0 and Rt in your forthcoming PR? and if so @yemarshall are you ok with deleting this sentence for now and then revisiting after @alavendelm's PR is opened?